### PR TITLE
Update ingress.md

### PR DESCRIPTION
**Pull request by guresonur**
'spec' is just another field that any Kubernetes object (Ingress here) needs.

### CODE REVIEW

**Review by divya-mohan0209 — COMMENTED** [review]
/lgtm

File: content/en/docs/concepts/services-networking/ingress.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ A minimal Ingress resource example:
 
 {{< codenew file="service/networking/minimal-ingress.yaml" >}}
 
-As with all other Kubernetes resources, an Ingress needs `apiVersion`, `kind`, and `metadata` fields.
+As with all other Kubernetes resources, an Ingress needs `apiVersion`, `kind`, `metadata` and `spec` fields.
```
Comment (tengqm) [inline content/en/docs/concepts/services-networking/ingress.md:77]: No. The `spec` field is pretty common among k8s resources, but there are still some resources that do not have a spec, for example , `Secret`.

Comment (guresonur) [reply content/en/docs/concepts/services-networking/ingress.md:77]: But it is required for Ingress.
Would this below not be more helpful?

```suggestion
An Ingress needs `apiVersion`, `kind`, `metadata` and `spec` fields.
```

**Review by sftim — COMMENTED** [review]
/lgtm

**Pull request merged**